<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Marker (average)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Stil, validiran podatak : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Boja, dobar flag : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="validanMarker"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="normalMarker"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Stil, sirov podatak :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Boja, los flag : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="bojaOK">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="bojaBAD">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Velicina markera : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="glavniMarkerSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="glavniMarkerAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Centralna linija (average)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Stil linije : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Sirina linije : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Boja : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="midlineStil"/>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="midlineWidth">
          <property name="minimum">
           <double>0.050000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="midlineAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="midlineBoja">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Ekstremi (min, max)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Izbor crtanja : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Stil markera : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Boja : </string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Velicina markera : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="ekstremCrtaj">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="ekstremBoja">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="ekstremMarker"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="ekstremSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="ekstremAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Fill</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Izbor crtanja : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Komponenta 1 : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Komponenta 2 :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Boja : </string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="fillKomponenta1"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="fillKomponenta2"/>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="fillAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="fillCrtaj">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="fillBoja">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Status warning</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="7" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Samo okolisni uvjeti boja:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="statusCrtaj">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="statusBoja">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="statusStil"/>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="statusSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDoubleSpinBox" name="statusAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.400000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Crtaj :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Stil markera :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Boja :</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Prozirnost :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Velicina markera :</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Samo okolisni uvjeti alpha:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QDoubleSpinBox" name="okolisStatusAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.400000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="okolisStatusBoja">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
